### ISSUE TITLE
Logging verbosity

### ISSUE THREAD

**Issue by rathbunr** [ansible-collections/microsoft.ad#69]
##### SUMMARY

I was doing some troubleshooting and another developer was inspecting your code.  It doesn't appear when you increase Ansible logging by raising the verbosity that the output is improved.
 
##### ISSUE TYPE
- Add improved logging verbosity levels for troubleshooting

##### COMPONENT NAME

Logging

##### ADDITIONAL INFORMATION

It would aid users who need to debug issues with the microsoft ad plugin

```yaml

```

**Comment by jborean93** [comment]
Which plugin in general? The modules don't have any changes in logging as they are executed on the remote side while I can see some benefits for the LDAP plugin I'm not 100% convinced it will help too much.

**Comment by rathbunr** [comment]
I think this can be closed, but I have encountered another issue but am unsure how to proceed here. I can replicate the issue on an AMD CPU based platform, but on the Intel platform I cannot. The intel CPU based ansible instance is able to perform an AD sync without issues, and I built both systems identically minus the CPU type. Describing the fault is difficult, because I am at a loss how to isolate the root cause.
